csvcubed/csvcubed/models/csvcubedexception.py

```diff
@@ -0,0 +1,281 @@
+from abc import ABC
+from enum import Enum
+from pathlib import Path
+
+from csvcubed.models.errorurl import HasErrorUrl
+
+
+class CsvcubedExceptionMsges(Enum):
+    """
+    Error messages for the exceptions thrown in csvcubed.
+    """
+
+    JsonldNotSupported = "The input csvw json-ld is not supported"
+
+    InputTypeUnknown = "The input type is unknown"
+
+    RDFGraphCannotBeNone = "RDF graph cannot be none"
+
+    CsvToDataFrameLoadFailed = "Failed to load csv to dataframe"
+
+    InvalidNumberOfRecords = "Expected {excepted_num_of_records} record(s), but found {num_of_records} record(s)"
+
+    FailedToReadSparqlQuery = (
+        "Failed to read sparql query from file at {sparql_file_path}"
+    )
+
+    InvalidCSVFilePath = "Currently, inspect cli only suports CSVs with local file paths. CSVs from HTTP urls are not yet supported"
+
+    FailedToLoadTableSchemaIntoRDFGraph = "An error occured while loading table schema '{table_schema_file}' into rdf graph"
+
+    FailedToParseJSONldtoRDFGraph = "An error occured while parsing CSV-W JSON-LD to RDF graph ({csvw_metadata_file_path})"
+
+    UnsupportedComponentPropertyType = "Property type {property_type} is not supported"
+
+    FailedToConvertDataFrameToString = "Failed to convert dataframe to string"
+
+    UnexpectedSparqlASKQueryResponseType = (
+        "Unexpected ASK query response type {response_type}"
+    )
+
+    UnexpectedSparqlASKQueryResults = (
+        "Unexpected number of results for ASK query {num_of_results}"
+    )
+
+
+class CsvcubedExceptionUrls(Enum):
+    """
+    Urls for the exceptions thrown in csvcubed.
+    """
+
+    JsonldNotSupported = "http://purl.org/csv-cubed/err/jsonld-not-supported"
```

Comment:
I take it these URIs are ones you want but still need to be created when you do the docs?

Reply:
Yes, that is correct